lib/persisted-model.js

```diff
@@ -993,15 +993,8 @@ module.exports = function(registry) {
         {verb: 'post', path: '/change-stream'},
         {verb: 'get', path: '/change-stream'},
       ],
-      accepts: {
-        arg: 'options',
-        type: 'object',
-      },
-      returns: {
-        arg: 'changes',
-        type: 'ReadableStream',
-        json: true,
-      },
+      accepts: {arg: 'options', type: 'object', http: 'optionsFromRequest'},
```

Comment:
Actually, on the second thought, considering that Operation hooks observers now expect that `ctx.options` is always constructed by the server and cannot be injected by the client, I think the original `options` configuration would create a serious security vulnerability and therefore we should change it even though it may break existing apps.